content/2016-10-16-haskell-servant-template.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+tags: haskell prometheus servant docker
+date: 2016-10-16
+title: servant-template: production-ready Haskell web services in 5 minutes
+author: jml
+---
+
+If you want to write a web API in Haskell, then you should start by using my
+new cookiecutter template at https://github.com/jml/servant-template. It'll
+get you a production-ready web service in 5 minutes or less.
+
+For some values of "production". And "ready".
```

Comment:
I've toned it down. Would rather no humour than humour that doesn't work. Thanks.